tools/function_finder/helpers.py

```diff
@@ -0,0 +1,39 @@
+# search for scratches with the name on decomp.me
+import json
+import requests
+
+
+def find_scratches(name):
+    try:
+        response = requests.get(f"https://decomp.me/api/scratch?search={name}")
+        response.raise_for_status()
+        scratches = json.loads(response.text)
+    except requests.exceptions.HTTPError as http_err:
+        return None
+    except Exception as err:
+        return None
+
+    best_result = None
+    best_percent = 0
+
+    for result in scratches["results"]:
+        if not "name" in result:
+            continue
+        # seems to give approximate matches, skip these
+        if result["name"] != name:
+            continue
+        if result["platform"] != "saturn":
```

Comment:
I think the platform needs to be a parameter?

Reply:
I am glad reviewing brings light to oversights like this. Fixed this and the other change to link the scraper result to the function list